core/bentley/src/ClassUtils.ts

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module ClassUtils
+ */
+
+/** Utility functions for working with JavaScript's concept of classes.
+ * @beta
```

Comment:
I would make these utility classes internal unless we have some reason we want 3rd parties to call them directly.

Reply:
done